py/server/deephaven/table_factory.py

```diff
@@ -76,14 +75,11 @@ def time_table(period: Union[Duration, int, str, datetime.timedelta, np.timedelt
     try:
         builder = _JTableTools.timeTableBuilder()
 
-        if not isinstance(period, str) and not isinstance(period, int):
-            period = time.to_j_duration(period)
-
-        builder.period(period)
+        if period:
```

Comment:
Confirming that an exception will be thrown if `period` is `None`.  An early impl had a fixed value, which was not good.  I _think_ this fixed value was removed everywhere, but I'm not 100%.

Reply:
Good call; there was an error thrown, but it wasn't as obvious in the java layer. I've explicitly accounted for this now and added a test case.